wordpress-mobile/WordPress-iOS
# Switch over showing/fetching Streak data to new architecture

This was the last Insight that was using the old architecture/backend. Now all of them are powered by the new data layer :)

~Note: I noticed (present also in current implementation) problem with displaying the graph for sites with extremely heavy posting traffic (like our team P2, for example). I filed a [bug in WPKit](https://github.com/wordpress-mobile/WordPressKit-iOS/issues/104) and will fix it next week.~ fixed it upstream in WPKit already: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/107

To test:

Verify the app builds
Verify the data shown for Streaks in stats insights is the same as above

Update release notes:
